features/employers/employer_attestation.feature

```diff
@@ -147,3 +147,29 @@ Feature: Employer Profile
     And Employer Staff clicks documents tab
     Then Employer Staff should see attestation status Rejected
     When Employer goes to the benefits tab I should see plan year information
+
+  Scenario: Employer Deletes submitted documents
+    Then Employer uploads an attestation document
+    And Employer clicks delete in actions
+    Then Employer should not see submitted document
+
+  Scenario: Employer should not delete submitted documents
```

Comment:
Employer should not be allowed to delete a document which is not in 'submitted' state. i.e, accepted, rejected or info needed. Please update scenario name accordingly.